scripts/gluster-block/gluster-block-basic.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# if anything fails, we'll abort
+set -e
+
+# install basic dependencies for building the tarball and srpm
+yum -y install git autoconf automake gcc libtool bison flex make mock
+# gluster repositories contain additional -devel packages
+yum -y install centos-release-gluster
+yum -y install libuuid-devel targetcli tcmu-runner glusterfs-api-devel
+
+# clone the repository, github is faster than our Gerrit
+#git clone https://review.gluster.org/glusterfs
+
+git clone --depth 2 https://github.com/gluster/gluster-block 
+cd gluster-block
+
+# by default we clone the master branch, but maybe this was triggered through a PR?
+if [ -n "${ghprbPullId}" ]
+then
+	git fetch origin pull/${ghprbPullId}/head:pr_${ghprbPullId}
+	git checkout pr_${ghprbPullId}
+fi
+
+# generate a version based on branch.date.last-commit-hash
```

Comment:
The versioning really does not matter, you're not building packages that get redistributed.